cockatrice/src/messagelogwidget.cpp

```diff
@@ -326,8 +326,8 @@ void MessageLogWidget::logMoveCard(Player *player,
     } else if (targetZoneName == deckConstant()) {
         if (newX == -1) {
             finalStr = tr("%1 puts %2%3 into their library.");
-        } else if (newX == targetZone->getCards().size() - 1) {
-            finalStr = tr("%1 puts %2%3 on bottom of their library.");
+        } else if (newX == targetZone->getCards().size()) {
```

Comment:
I think just having a >= is appropriate here, while the game allows you to put a card a million cards from the top, this is the result anyway :)